cmd/agent-sandbox-controller/main.go

```diff
@@ -42,12 +46,19 @@ func main() {
 	var enableLeaderElection bool
 	var probeAddr string
 	var extensions bool
+	var enablePprof bool
+	var enablePprofDebug bool
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
 	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
 	flag.BoolVar(&extensions, "extensions", false, "Enable extensions controllers.")
+	flag.BoolVar(&enablePprof, "enable-pprof", false,
+		"Enable pprof profiling endpoint (/debug/pprof/profile) on the metrics server.")
```

Comment:
Clarification: Consider changing the description to "Enable CPU profiling endpoint..." to be precise, as `enable-pprof-debug` enables the other types (memory, block, etc).